networking/metallb/metallb-operator-install.adoc

```diff
@@ -33,8 +26,8 @@ include::modules/nw-metallb-operator-limit-speaker-to-nodes.adoc[leveloffset=+2]
 [role="_additional-resources"]
 .Additional resources
```

Comment:
Since this "Additional resources" section is at the end of the assembly, you could consider changing this to be an H1 heading and include an anchor ID.